locale/fr/security.md

```diff
@@ -29,7 +29,7 @@ Les détails de fonctionnement de ce processus se trouvent dans le [dépôt du _
 
 Merci de participer à l'amélioration de la sécurité de Node.js et de son écosystème. Vos efforts et un signalement responsable sont fortement appréciés et seront reconnus.
 
-* Le signalement est reçu et assigné à un·e responsable. Cette personne va coordonner la production et la livraison du correctif. Le problème est confirmé et une liste des versions affectées est déterminée. Le code est audité afin de repérer d'éventuels problèmes similaires. Les correctifs sont préparés pour chaque branche de Node.js encore en maintenance. Ces correctifs ne sont pas directement inclus dans le dépôt public et sont mis de côté en attendant une annonce publique.
+* Le signalement est reçu et assigné à un responsable. Cette personne va coordonner la production et la livraison du correctif. Le problème est confirmé et une liste des versions affectées est déterminée. Le code est audité afin de repérer d'éventuels problèmes similaires. Les correctifs sont préparés pour chaque branche de Node.js encore en maintenance. Ces correctifs ne sont pas directement inclus dans le dépôt public et sont mis de côté en attendant une annonce publique.
```

Comment:
Pourquoi ce changement?

Reply:
On dit un responsable. De plus, il y avait un ·

Reply:
Le point médian est là pour permettre de genrer l'article au féminin et au masculin à la fois :) let's keep it that way.